config/locales/forms/main_people_forms/en.yml

```diff
@@ -10,13 +10,12 @@ en:
         partnership: Partner details
         soleTrader: Business owner details
       description_1:
-        localAuthority: Provide the name and date of birth for each chief executive of this local authority or public body.
-        limitedCompany: Provide the name and date of birth for each director of this business.
-        limitedLiabilityPartnership: Provide the name and date of birth for each partner of this business.
-        overseas: Provide the name and date of birth for each owner of this business or organisation.
-        partnership: Provide the name and date of birth for each partner of this business.
-        soleTrader: Provide the name and date of birth for the owner of this business.
-      description_2: Add their details one person at a time. If you need to add more people, select 'Add another person'.
+        localAuthority: Enter the name and date of birth for each chief executive of the business.
```

Comment:
"Chief executive of the business" sounds a bit odd, as a local authority isn't a business – should this be "chief executive of this local authority or public body"?

Reply:
That's what Rose requested, I'll double check.

Reply:
She's going to check with Martin on Friday, can we leave it as is for now?